src/wallet/wallet2.h

```diff
@@ -67,6 +67,7 @@
 #define MONERO_DEFAULT_LOG_CATEGORY "wallet.wallet2"
 
 class Serialization_portability_wallet_Test;
+class Wallet_Accessor_Test;
```

Comment:
Caps and underscores. That's some new naming scheme. The above class isn't used anywhere fwiw, it can be removed.